app/views/recipes/show.html.erb

```diff
@@ -18,7 +18,7 @@
       </p>
       <p class="center">
         <small>
-          <%= @recipe.chef.name.capitalize %>
+          <%= @recipe.chef.chefname.capitalize %>
```

Comment:
"chefname" it's no longer the name of the column, you need to change it back to "name" because this generates an issue when trying to create a new recipe (As seen in the image below)

![Screenshot (220)](https://user-images.githubusercontent.com/105248457/170847967-7b0139a4-3ba5-4428-b537-8b1377b80218.png)

Reply:
Donde